README.md

```diff
@@ -25,3 +25,52 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 ## Further help
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+
+## Features and Benefits
+
+## Calendar HeatMap
+
+The Angular calendar HeatMap visualizes time series data, with each data point representing a value bound to a specific time.
+
+## Bubble HeatMap Chart
+
+The Angular [bubble HeatMap](https://ej2.syncfusion.com/angular/documentation/heatmap-chart/bubble-heatmap/?utm_source=github&utm_medium=listing&utm_campaign=angular-heatmap-chart-github-samples) Chart, or the matrix bubble chart, visualizes data using variations in bubble attributes such as size, color, and sector.
```

Comment:
[Angular Bubble HeatMap] (https://ej2.syncfusion.com/angular/documentation/heatmap-chart/bubble-heatmap?utm_source=github&utm_medium=listing&utm_campaign=angular-heatmap-chart-github-samples)

Reply:
Updated

Reply:
This not yet resolved..recheck

Reply:
Resolved